src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -161,6 +194,8 @@ export default {
         ],
       },
       ccsAnalysisOutputOptions: ['Yes', 'No'],
+      regexToMatch: /(\d{0}\.\d{1,2})$/,
```

Comment:
Not sure I understand the `\d{0}`.  The zero means don't try to match any digits, which is the same as leaving that out of the regex entirely.

Reply:
Also, do you really want to fail to match values that have more than 2 decimal digits?  You can see this regex in action here:  https://regex101.com/r/p8kbkf/2

Note how it matches only the decimal point and two digits.  It also matches the last section where more than one decimal place was used and it allows alphabet characters before the final decimal section.  So this may not do what you think it is doing.